### PR TITLE
Click cards and attack once

### PR DESCRIPTION
**Pull request by majecty**

### CODE REVIEW

File: Assets/Scripts/Battle/Turn/PlayerTurn.cs [hunk 1]
```diff
@@ -11,13 +11,14 @@ public class PlayerTurn : Turn
...
-                Debug.Log("User clicked " + _clickedCardIndex);
+            Run<Unit> _waitingUserClickTurnEnd = WaitingUserClickTurnEnd();
+
+            _waitingUserClickTurnEnd.Then((_unit) => {
+                Debug.Log("User clicked " + mState.clickedCardIndexes.DebugToString());
                 return Run<Unit>.Default();
             });
 
-            Run<Unit> _attack = _userClickedCard.Then((_clickedCardIndex) => Attack(_clickedCardIndex));
+            Run<Unit> _attack = _waitingUserClickTurnEnd.Then((_clickedCardIndexes) => Attack());
```
Comment (remagpie) [inline Assets/Scripts/Battle/Turn/PlayerTurn.cs]: _clickedCardIndexes가 사용되지 않음

File: Assets/Scripts/Battle/UI/Card.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        private void Start()
+        {
+            mButton = GetComponent<Button>();
+            mButton.onClick.AddListener(() => Battle.Instance.EventReceiver.OnClickCard(cardIndex));
+        }
+
+        private void Update()
+        {
+            var _state = Battle.Instance.State;
+            var isClicked = _state.clickedCardIndexes.Contains(cardIndex);
```
Comment (remagpie) [inline Assets/Scripts/Battle/UI/Card.cs]: underbar 넣을것

File: Assets/Scripts/Battle/UI/Card.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using System.Collections;
+
+namespace Battle.UI
+{
+    [RequireComponent (typeof (Button))]
+    public class Card : MonoBehaviour
+    {
+        [SerializeField] Image selectedImage;
```
Comment (remagpie) [inline Assets/Scripts/Battle/UI/Card.cs]: private 넣을것

File: Assets/Scripts/Battle/Turn/PlayerTurn.cs [hunk 4]
```diff
@@ -11,13 +11,14 @@ public class PlayerTurn : Turn
```
Comment (remagpie) [inline Assets/Scripts/Battle/Turn/PlayerTurn.cs:12]: line 10 : eventReceiver -> _eventReceiver

File: Assets/Scripts/Battle/Turn/PlayerTurn.cs (hunk 1 above)
Comment (majecty) [inline Assets/Scripts/Battle/Turn/PlayerTurn.cs]: 수정함

File: Assets/Scripts/Battle/UI/Card.cs (hunk 2 above)
Comment (majecty) [inline Assets/Scripts/Battle/UI/Card.cs]: 수정함.

File: Assets/Scripts/Battle/UI/Card.cs (hunk 3 above)
Comment (majecty) [inline Assets/Scripts/Battle/UI/Card.cs]: 수정.

**Pull request merged**